OCPBUGS#17371: Wipe disks before deploying LVMS on private CI environment (#69930)
openshift/openshift-docs: modules/deploying-lvms-on-sno-cluster.adoc

[id="lvms-deployment-requirements-for-sno-ran_{context}"]
== Requirements

Before deploying {lvms} on {sno} clusters, ensure that the following requirements are met:

* You have installed {rh-rhacm-first} on an {product-title} cluster.
* Every managed cluster has dedicated disks that are used to provision storage.

> Before deploying {lvms} on a private CI environment, where virutal machines and storage devices can be reused, ensure that you have wiped the disks that are not in use. 

Reviewer: Typo "virutal". Plus @agantony had mentioned an improvement that I wasn't sure if you saw/considered. I think it's a good suggestion that puts it more in active voice.

Posting here to fix the typo and use his suggestion, except putting it back to "wiped".

```suggestion
Before deploying {lvms} in a private CI environment where you can reuse virtual machines and storage devices, ensure that you have wiped the disks that are not in use.
```

Author: @bergerhoffer Thank you for pointing this out. I failed to notice this suggestion during the peer review. Updated the content based on the suggestion. PTAL.
Thanks.

> Before deploying {lvms} on a private CI environment, where virutal machines and storage devices can be reused, ensure that you have wiped the disks that are not in use. 
> [NOTE]
> ====
> You can wipe only the disks that are not in use.
> ====

Reviewer 2: ```suggestion
Before deploying {lvms} in a private CI environment where you can reuse virtual machines and storage devices, ensure that you have deleted the disks that are not in use.

[NOTE]
====
You can only delete the disks that are not in use.
====
```

Author: @agantony 
Thank you for the review. Just one query on the comment-

When we say "You can delete only the disks that are not in use" - the limitation is on the type of devices that can be deleted.
In case of  "You can only delete the disks that are not in use" - the limitation is the action of deletion.

Or to avoid confusion we can rephrase it to - "You cannot delete the disks that are in use"
Any comments on this approach?

@suleymanakbas91 Could you please confirm if there is a technical difference between "deleting" and "wiping" the disks.

Reviewer 2: "You cannot delete the disks that are in use" --> This SGTM!

Reviewer 3: `wipe` is the common usage, so I'd stick to that.
